recipes/libcurl/all/conanfile.py

```diff
@@ -337,38 +396,39 @@ def _build_with_cmake(self):
         cmake.build()
 
     def package(self):
-        self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
+        with tools.chdir(self.source_folder):
+            self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
 
-        # Execute install
-        if self.settings.compiler != "Visual Studio":
-            env_run = RunEnvironment(self)
+            # Execute install
+            if self.settings.compiler != "Visual Studio":
+                env_run = RunEnvironment(self)
 
-            with tools.environment_append(env_run.vars):
-                with tools.chdir(self._source_subfolder):
-                    autotools, autotools_vars = self._configure_autotools()
-                    autotools.install(vars=autotools_vars)
-        else:
-            cmake = self._configure_cmake()
-            cmake.install()
+                with tools.environment_append(env_run.vars):
+                    with tools.chdir(self._source_subfolder):
+                        autotools, autotools_vars = self._configure_autotools()
+                        autotools.install(vars=autotools_vars)
+            else:
+                cmake = self._configure_cmake()
+                cmake.install()
 
-        if self._is_mingw:
-            # Handle only mingw libs
-            self.copy(pattern="*.dll", dst="bin", keep_path=False)
-            self.copy(pattern="*dll.a", dst="lib", keep_path=False)
-            self.copy(pattern="*.def", dst="lib", keep_path=False)
-            self.copy(pattern="*.lib", dst="lib", keep_path=False)
-
-        self.copy("cacert.pem", dst="res")
-
-        # no need to distribute share folder (docs/man pages)
-        shutil.rmtree(os.path.join(self.package_folder, 'share'), ignore_errors=True)
-        # no need for pc files
-        shutil.rmtree(os.path.join(self.package_folder, 'lib', 'pkgconfig'), ignore_errors=True)
-        # no need for cmake files
-        shutil.rmtree(os.path.join(self.package_folder, 'lib', 'cmake'), ignore_errors=True)
-        # Remove libtool files (*.la)
-        if os.path.isfile(os.path.join(self.package_folder, 'lib', 'libcurl.la')):
-            os.remove(os.path.join(self.package_folder, 'lib', 'libcurl.la'))
+            if self._is_mingw:
+                # Handle only mingw libs
+                self.copy(pattern="*.dll", dst="bin", keep_path=False)
+                self.copy(pattern="*dll.a", dst="lib", keep_path=False)
+                self.copy(pattern="*.def", dst="lib", keep_path=False)
+                self.copy(pattern="*.lib", dst="lib", keep_path=False)
+
+            self.copy("cacert.pem", dst="res")
+
+            # no need to distribute share folder (docs/man pages)
+            shutil.rmtree(os.path.join(self.package_folder, 'share'), ignore_errors=True)
```

Comment:
```suggestion
            tools.rmdir(os.path.join(self.package_folder, 'share'))
```

Use `tools.rmdir` instead of `rmtree` for every condition